server/wizard/steps/step_webhook_configuration.go

```diff
@@ -0,0 +1,44 @@
+package steps
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow/steps"
+
+	"github.com/mattermost/mattermost-plugin-zoom/server/config"
+)
+
+const (
+	stepNameWebhookConfiguration = "webhook_configuration"
+
+	stepTitleWebhookConfiguration = "Configure webhook in Zoom"
+
+	stepDescriptionWebhookConfiguration = `1. Click on the **Feature** category in the left sidebar.
+2. Enable **Event Subscriptions**.
+3. Click **Add New Event Subscription** and give it a name \(e.g. "Mattermost events"\).
+4. Enter in **Event notification endpoint URL**: %s
+5. For the **Event notification receiver** field, select "Only users installed this app" TODO: Is this right?
```

Comment:
Very dangerous TODO :sweat_smile: (it may slip into the final version)
Do you mind investigating what are the implications of marking/not marking this field and share with @aaronrothschild so he can make a decision on wording?